cmd/gokr-packer/packer.go

```diff
@@ -165,6 +175,42 @@ func findFlagFiles() (map[string]string, error) {
 	return contents, nil
 }
 
+func findBuildArgsFiles() (map[string]string, error) {
+	buildArgsFilePaths, err := findPackageFiles("buildargs")
```

Comment:
Here and elsewhere: this should be `buildflags` (see e.g. `go help build`, which calls them flags, not args)